.circleci/config.yml

```diff
@@ -1,8 +1,11 @@
 version: 2
-publish: &publish
-  working_directory: ~/my-app
+job_setup: &job_setup
```

Comment:
we could change this with something named like build that also does the `checkout` and `npm install` steps, as they seem to be repeated on each job

Reply:
That sounds much better

Reply:
I renamed job_setup to container_setup, but I've decided to abandon reusing the build steps of now as I was having difficulty with merging nested steps and objects.